spec/lumberjack_spec.rb

```diff
@@ -97,7 +99,7 @@
   context "when compiled from source" do
     let(:lsf) do
       # Start the process, return the pid
-      IO.popen(["./logstash-forwarder", "-config", config_file, "-quiet"])
+      IO.popen(["./logstash-forwarder", "-config", config_file, "-quiet", "-from-beginning=true"])
```

Comment:
Maybe I had an old logstash binary? This caused issues when I ran the tests locally.